kuksa-client/kuksa_client/grpc/aio.py

```diff
@@ -318,6 +318,11 @@ async def subscribe(self, *,
 
     @check_connected_async
     async def authorize(self, token: str, **rpc_kwargs) -> str:
```

Comment:
Should we possibly when changing here also add a `*, `after `self` to make it identical to the same method in ` __init__.py` and similar to the other methods in this file?

Reply:
My question would be _why_ `*` is included in those other function signatures...

Reply:
Fair point - as long as it is consistent I am happy.